### ISSUE TITLE
wifi mac address always change

### ISSUE THREAD

**Issue by jhonoryza** [RDA8810/linux-RDA8810#25]
everytime reboot wifi mac address always change, is it a bug?

**Comment by txurtxil** [comment]
yes, i think that u-boot form xunlong correct this issue https://github.com/OrangePiLibra/OrangePiRDA_uboot

**Comment by jhonoryza** [comment]
is that xunglong u-boot have also a nand-boot?

**Comment by txurtxil** [comment]
Today no, but we can update aib u-boot sources to fix this problem, we can told him
